src/services/ClaudyActionComponents/Support.jsx

```diff
@@ -14,13 +21,21 @@ export class Support extends Component {
   }
 
   componentWillReceiveProps (nextProps) {
-    nextProps.opened
+    // if email from instance and input not already filled
+    if (!this.state.email && nextProps.instanceData) {
+      const email = nextProps.instanceData.attributes.email
+      this.setState({
+        email,
+        isEmailValid: isValidEmail(email)
+      })
+    }
+    nextProps.opened !== this.props.opened && nextProps.opened
       ? this.onOpen() : this.onReturn()
   }
 
   resize () {
     typeof this.props.resizeIntent === 'function' && this.props.resizeIntent({
-      height: 27.5 * 16 // 27.5em
+      height: (31.5 * 16) // 31.5em
```

Comment:
I think that doesn't cost anything here and it will easily force us to resize only using `em` here. We won't be attempted to use pixel directly. What do you think?